vtex/instore-pickup-points-modal
# Fix/unavailable pickup points

**Run verdaccio to test**

1. Start verdaccio: docker run -it --rm --name verdaccio -p 4873:4873 verdaccio/verdaccio

1. Create any user into verdaccio: npm adduser --registry http://localhost:4873

1. Runs: yarn prepublishOnly && yarn publish --registry http://localhost:4873  --new-version "0.0.1-verdaccio"

1. Go to http://localhost:4873/, you can check for the new publish package

1. Install in chekcout-instore: YARN_REGISTRY=http://localhost:4873/ yarn add your-package-name@0.0.1-verdaccio
